src/entity-manager/EntityManager.ts

```diff
@@ -708,7 +708,7 @@ export class EntityManager {
             throw new Error(`Value "${value}" is not a number.`);
 
         return this
-            .createQueryBuilder(entityClass, "entity")
+            .createQueryBuilder<any>(entityClass, "entity")
```

Comment:
Use same construction as in `increment` method? `} as QueryDeepPartialEntity<Entity>`